docs/conf.py

```diff
@@ -171,7 +171,7 @@
 # Add any extra paths that contain custom files (such as robots.txt or
 # .htaccess) here, relative to this directory. These files are copied
 # directly to the root of the documentation.
-html_extra_path = ["public"]
+html_extra_path = ["robots.txt"]
```

Comment:
@RealOrangeOne Is this valid? The [sphinx docs](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_extra_path) say

> A list of **paths that contain extra files** not directly related to the documentation, such as robots.txt or .htaccess.

which I'd interpret as meaning the files _within_ these paths are copied to the root - obviously `robots.txt` doesn't have files within it. Any reason this shouldn't remain unchanged (with robots.txt left in public/)?

Reply:
I did it to restore it back to as it was before #9135. I agree it's probably better to leave it in this new way, which is more correct.